app/helpers/auth_helper.rb

```diff
@@ -0,0 +1,9 @@
+module AuthHelper
+  def current_user
+    if session[:user_id]
+      return User.find(session[:user_id])
+    else
+      return nil
+    end
```

Comment:
This isn't needed because you have current_user in the application controller. Also, it could be a single line
```User.find_by(id: sesssion[:user_id]) if session[:user_id]```

else return nil is not often needed in the ruby